### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Files_Add Database File_DevTools: Programmatic name is not given for "more option" button present beside "path" edit filed. 

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#25129]
**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification"**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
## Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG4.1.2; #FTP; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio; #Win32; #Devtools; #NonBenchmark; #NameRoleValue; #Closed;
## Environment Details:
Application Name: Azure Data Studio
Operating System: Win11
Azure data studio 1.47.0
## Repro Steps:
1. Install Azure data studio application and open it
3. Make sure to enable "Preview" feature in settings.
4. Tab till "Database" and press enter to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press enter.
7. Tab till "Files" tab and press enter.
8. Tab till "Add" button and press enter. Add database file dialog will open.
9. Tab till "more option" button present beside path edit filed and inspect it.
10. You will observe that Programmatic name is not given for button.
## Actual Result:
Programmatic name is not given for "more option" button present beside "path" edit filed. 
## Expected Result:
Programmatic name should be given for "more option" button present beside "path" edit filed. 
## User Impact:
Screen reader user will not get the purpose of button and speech input user will be unable to interact with button. 
### Attachment:
![25129_A11y_Azure Data Studio_Nov2023_Files_Add Database File_DevTools](https://github.com/microsoft/azuredatastudio/assets/124254981/6cf23cf3-bd2e-4dfe-92de-211c075bf63f)

**Comment by corivera** [comment]
@kapilvaishna We added a better name for the specified button control. Here's a screenshot:
![BrowseFilesButton](https://github.com/microsoft/azuredatastudio/assets/12820011/ef09fcf4-6962-464b-b140-981af32a8426)

**Comment by kapilvaishna** [comment]
@corivera  could you share the version where this update is available to verify.

**Comment by corivera** [comment]
@kapilvaishna The latest Insiders build for Windows can be found [here](https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/stable)

**Comment by kapilvaishna** [comment]
Issue is not repro in Azure data studio insider (Version 1.48.0)  
![Screenshot (993)](https://github.com/microsoft/azuredatastudio/assets/124254981/d625625d-0118-4f5d-9be8-a8930206538f)
